crdt/gcounter.py

```diff
@@ -0,0 +1,35 @@
+import uuid
+
+class GCounter:
+    """
+    GCounter implements a grow-only counter CRDT. It must be instantiated with a
+    network-unique ID.
+    """
+
+    def __init__(self, id=uuid.uuid4()):
```

Comment:
Learned something new about Python, apparently the `uuid.uuid4()` here is only executed once so `GCounter()` will always produce the same id.